docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,304 @@
+
+### Metrics
+
+In this section we analyse key metrics extracted from Prometheus data on the 200-nodes experiment
+with configuration `c=1,r=400`.
+
+#### Mempool size
+
+The following figures show the evolution of the average and maximum mempool size over all full
+nodes. 
+
+**Average size** On `v1`, the average mempool size mostly stays below 1000 outstanding transactions
+except for a peak above 2000, coinciding with the moment the system reached round number 1 (see
+below). For these particular runs, this result is better than the baseline, which oscilates between
+1000 and 2500.
+
+With latency emulation, the average mempool size stays mostly above 2000 outstanding transactions
+with peaks almost reaching the maximum mempool size of 5000 transactions.
+
+| v0.38 | v1 (without LE / with LE) 
+| :--------------:|:--------------:|
+| ![mempool-avg-baseline](img38/200nodes/avg_mempool_size_ylim.png) | ![mempool-avg](imgs/v1/200nodes/metrics/avg_mempool_size.png)
+| | ![mempool-avg-le](imgs/v1/200nodes_with_latency_emulation/metrics/avg_mempool_size.png)
+
+**Maximum size** The maximum mempool size indicates when one or more nodes have reached their
+maximum capacity in terms of the number of transactions they can hold. In version `v0.38`, it's
+apparent that most of the time, at least one node that is dropping incoming transactions.
+Conversely, in `v1`, this happens less often, especially after reaching round 1 (as detailed below).
+
+However, when we introduce latency emulation into `v1`, there is consistently at least one node with
+a saturated mempool.
+
+| v0.38 | v1 (without LE / with LE)
+| :--------------:|:--------------:|
+| ![mempool-cumulative-baseline](img38/200nodes/mempool_size_max.png) | ![mempoool-cumulative](imgs/v1/200nodes/metrics/mempool_size_max.png)
+| | ![mempoool-cumulative-le](imgs/v1/200nodes_with_latency_emulation/metrics/mempool_size_max.png)
+
+#### Peers
+
+On all expertiments, the number of peers was stable on all nodes. As expected, the seed nodes have
+more peers (around 125) than the rest (between 20 and 70 for most nodes). The red dashed line
+denotes the average value. Just as in the baseline, the fact that non-seed nodes reach more than 50
+peers is due to [\#9548].
+
+| v0.38 | v1 (without LE / with LE) 
+|:--------------:|:--------------:|
+| ![peers](img38/200nodes/peers.png) | ![peers](imgs/v1/200nodes/metrics/peers.png)
+| | ![peers](imgs/v1/200nodes_with_latency_emulation/metrics/peers.png)
+
+#### Consensus rounds
+
+On both versions, most blocks took just one round to reach consensus, except for a few cases when it
+was needed a second round. On these two particular runs, we observe that `v0.38` required an extra
+round on more occasions than `v1`.
+
+With latency emulation, the performance is notably worse: the consensus module requires an extra
+round more often, even needing four rounds to finalise a block.
```

Comment:
```suggestion
round more often, even needing four rounds to finalise a block. This indicates that the values of consensus timeouts should be increased, so that to represent the actual delays in the network.
```